plugins/GSdx/GSRenderer.cpp

```diff
@@ -274,7 +287,7 @@ bool GSRenderer::Merge(int field)
 
 		if(m_regs->SMODE2.INT && m_interlace > 0)
 		{
-			if (m_interlace == 7 && m_regs->SMODE2.FFMD == 1) // Auto interlace enabled / Odd frame interlace setting
+			if(m_interlace == 7 && m_regs->SMODE2.FFMD == 1) // Auto interlace enabled / Odd frame interlace setting
```

Comment:
I don't get your logic. ``if (variable)`` basically just checks whether the variable is non-null. Why would anyone think that it should only be used for ``bool`` ? Either way, I don't want the discussion to progress off-topic and more importantly not in a subjective manner ;)

As stated already, The ``==`` is just useless. (FFMD is only one bit)